tests/_test_local/_test_web.py

```diff
@@ -34,17 +34,13 @@ def test_get_email_passwd_auth_key_ok(self, set_authentication_config):
 
 def test_webapi_0_run(tmp_path):
     """test complete run"""
-    _ = web.run(
-        simulation=sim_original, task_name="test_webapi", path=str(tmp_path / "sim_data.hdf5")
-    )
+    _ = web.run(stub=sim_original, task_name="test_webapi", path=str(tmp_path / "sim_data.hdf5"))
```

Comment:
Can't we keep the name `simulation` for the arguments in the web API? `stub` doesn't indicate what the argument really is, and `simulation` should work for all kinds of simulation types in the future, I guess.

Reply:
yea we agreed it should be `simulation`, I think this test maybe never got reverted (and was maybe missed since it's a local test). @magiWei could you change the `_test_web` to use `simulation` instead of `stub` as the kwarg? thanks.

Reply:
Modified.